File: Php/src/main/Event.php
<?php

class Event
{
    public $id;
    public $name;
    public $date;
    public $place;
    public $createdBy;
    public $imageUrl;
    public $shortInfo;
    public $description;

    /**
     * Event constructor.
     * @param Number $id
     * @param String $name
     * @param DateTime $date
     * @param $place
     * @param Number $createdBy
     * @param String $imageUrl
     * @param $shortInfo
     * @param $description
     */
    public function __construct($id, $name, $date, $place, $createdBy, $imageUrl, $shortInfo, $description)
    {
        $this->id = $id;
        $this->name = $name;
        $this->date = $date;
        $this->createdBy = $createdBy;
        $this->imageUrl = $imageUrl;
        $this->shortInfo = $shortInfo;
        $this->description = $description;
        $this->place = $place;
    }

    public function getMeta()
    {
        return $this->date->format("d/m/Y") . ", " . $this->place;
    }
}

File: Dotnet/Entities/Event.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Shop.Entities
{
    [Table("events")]
    public class Event
    {
        [Required] [Key] public int Id { get; set; }
        [Required] [StringLength(255)] public string Name { get; set; }
        [Required] public double Price { get; set; }

        [Required] [StringLength(255)] public string Place { get; set; }

        [Column("date")] public DateTime Date { get; set; }
        [Column("short_info")] public string ShortInfo { get; set; }
        [Column("image_url")] public string ImageUrl { get; set; }

        [Column("number_of_available_tickets")]
        public int NumberOfAvailableTickets { get; set; }

        [Column("created_by")]
        [StringLength(50)]
        public string CreatedBy { get; set; }

        [Column("age_range")]
        [StringLength(50)]
        public string AgeRange { get; set; }

        [Column("additional_info")]
        [StringLength(255)]
        public string AdditionalInfo { get; set; }
    }
}